templates/pgbouncer/pgbouncer-networkpolicy.yaml

```diff
@@ -36,6 +36,12 @@ spec:
           tier: monitoring
           component: prometheus
           release: {{ .Values.platform.release }}
+    - namespaceSelector:
+        matchLabels:
+          team: operations
+      podSelector:
+        matchLabels:
+          app: keda-operator
```

Comment:
I agree with your assessment in slack that this is not secure because there can be other namespaces with this label. I would like to see the following outcome from this PR:

- by default, the minimum network policy is applied to pgbouncer in airflow-chart installation, only allowing access from the workload(s) in this chart that require access.
- There is an optional configuration in values.yaml of type 'list' to add additional ingress rules to the network policy: pgbouncer.networkpolicy.ingress.extraRules

This includes removing the rule to enable prometheus scrape configuration from this chart.